### PR TITLE
describe the format of bhnt on the landing page

### PR DESCRIPTION
**Pull request by k-nut**
This is the first draft for #220. 

Feel free to change the wording or add your input if this should be positioned somewhere else on the page 🙂

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -14,8 +14,17 @@
     <header>
       <div class="container">
         <h1>{{ site.name | default: site.github.repository_name }}</h1>
-        <h2>{{ site.description | default: site.github.project_tagline }}</h2>
+	<h2> Berlin Hack and Tell is a monthly meetup for creative hackers, designers and artists</h2>
```
[the remark this answers is not in the document]
Comment (vog) [reply _layouts/default.html]: Proposal:

| ... for creative hackers, including coders, designers and artists.

File: _layouts/default.html [hunk 2]
```diff
@@ -14,8 +14,17 @@
...
 
+	<p> The rules are simple: On every last Tuesday of the month, there are eight slots for people that want
```
[the remark this answers is not in the document]
Comment (stefanhoth) [reply _layouts/default.html]: That's fine, we usually skipped it in December as even if you're doing it a week earlier you got to wrangle with Christmas.

Are there any other months in which you see a problem?

File: _layouts/default.html [hunk 3]
```diff
@@ -14,8 +14,17 @@
...
+	    to present their hacks. Every presenter has 5 minutes to present their hack, followed by five more minutes of questions 
+	    from the audience. The realm of what people can present is very open (e.g. your software side project or your latest
+	    hardware hack), we just ask presenters to follow our code of conduct and not pitch their startup but instead use this
+	    as a space to show all the silly things that they cannot present anywhere else. <br />
+	    Think of it like a sort of standup-hacking. </p>
+	<p> If this sounds fund to you, check our some of the past project below and visit our meetup page. We are looking forward to seeing you
+            at the next Berlin Hack and Tell! </p>
+
```
Comment (lucasrangit) [inline _layouts/default.html]: There appears to be a mix of tabs and spaces. I think this file is spaces only.

File: _layouts/default.html [hunk 4]
```diff
@@ -14,8 +14,17 @@
     <header>
       <div class="container">
         <h1>{{ site.name | default: site.github.repository_name }}</h1>
-        <h2>{{ site.description | default: site.github.project_tagline }}</h2>
+	<h2> Berlin Hack and Tell is a monthly meetup for creative hackers, designers and artists</h2>
 
+	<p> The rules are simple: On every last Tuesday of the month, there are eight slots for people that want
+	    to present their hacks. Every presenter has 5 minutes to present their hack, followed by five more minutes of questions 
+	    from the audience. The realm of what people can present is very open (e.g. your software side project or your latest
+	    hardware hack), we just ask presenters to follow our code of conduct and not pitch their startup but instead use this
```
Comment (lucasrangit) [inline _layouts/default.html]: Add link to our code of conduct.

File: _layouts/default.html [hunk 5]
```diff
@@ -14,8 +14,17 @@
...
+	    as a space to show all the silly things that they cannot present anywhere else. <br />
+	    Think of it like a sort of standup-hacking. </p>
+	<p> If this sounds fund to you, check our some of the past project below and visit our meetup page. We are looking forward to seeing you
```
Comment (lucasrangit) [inline _layouts/default.html]: fund -> fun

File: _layouts/default.html [hunk 6]
```diff
@@ -14,8 +14,17 @@
     <header>
       <div class="container">
         <h1>{{ site.name | default: site.github.repository_name }}</h1>
-        <h2>{{ site.description | default: site.github.project_tagline }}</h2>
```
Comment (lucasrangit) [inline _layouts/default.html]: Instead of removing ignoring `site.description` why not update it?

**Comment by lucasrangit** [comment]
@k-nut I summarized your text to be more concise. See if you can come up with something simpler that can be a one line description, like the following.

Berlin Hack and Tell is a monthly meetup where eight people present their hacks. You have five minutes followed by up to five minutes of questions. Show us code, not a slide deck or sales pitch! Follow our Code of Conduct and show off a silly/hobby project from your GitHub that you cannot present elsewhere. Think of it like "stand up hacking". Check out the past project below and sign up on our Meetup page!

**Comment by ligi** [comment]
like it - just instead of:

> Check out the past project below

> Check out the past projects below

**Comment by k-nut** [comment]
@lucasrangit wohoo, that is a really nice way of putting it! I pushed another commit that changes this. 🚀

Our Code of Conduct is the one from the hackandtell.org, right? Or do we have a Berlin version of it?

**Pull request merged**